src/site/content/en/blog/granular-chunking-nextjs/index.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Granular chunking in Next.js and Gatsby
```

Comment:
```suggestion
title: Improve Next.js and Gatsby page load performance with granular chunking
```
$X with $Y is a good format for documentation titles, where $X is the generic task that the user is trying to solve and $Y is the specific feature. The current title is focused on the feature, and users might not know why they should care about that feature. This format also improves the page's SEO because it captures both the generic task audience and the audience that is looking for information on the specific feature

Reply:
Great point thank you. I've updated it to say `Improved Next.js and Gatsby page load performance with granular chunking` since this isn't necessarily a change developers will need to do themselves when using Next.js or Gatsby (it's already shipped by default).